test/contact-summary.spec.js

```diff
@@ -126,4 +116,14 @@ describe('getContactSummary', () => {
       expect(error.message).to.contain('_id');
     }
   });
+
+  it('Cht API end-to-end test', async () => {
+    const chtScriptApi  = await harness.core.ChtScriptApi;
```

Comment:
this is a unit test. an end to end test is from the perspective of a user. 

can you look at adding a test which has a contact-summary javascript file which uses the `cht.v1` for something?

Reply:
@kennsippell, for sure, but this week I am mostly focused on OCA workflow. 
In order to do not block other tickets that are pending because of this we can merge this PR if it is OK, then I can add a test which has a contact-summary javascript file which uses the cht.v1 in another ticket.